stl/inc/thread

```diff
@@ -241,6 +242,74 @@ struct hash<thread::id> {
         return _Hash_representation(_Keyval._Id);
     }
 };
+
+#if _HAS_CXX20
+class jthread : private thread {
+public:
+    using id                 = thread::id;
+    using native_handle_type = thread::native_handle_type;

```

Comment:
I don't think we should provide this typedef given that the existing one doesn't provide anything useful and we need not provide it because it's existence is implementation defined.